public/prism.css

```diff
@@ -0,0 +1,417 @@
+  color: light-dark(hsl(230, 8%, 24%), hsl(220, 14%, 71%));
+}
+
+.language-json .token.null.keyword {
+  color: light-dark(hsl(35, 99%, 36%), hsl(29, 54%, 61%));
+}
+
+/* MD overrides */
+.language-markdown .token.url,
+.language-markdown .token.url>.token.operator,
+.language-markdown .token.url-reference.url>.token.string {
+  color: light-dark(hsl(230, 8%, 24%), hsl(220, 14%, 71%));
+}
+
+.language-markdown .token.url>.token.content {
+  color: light-dark(hsl(221, 87%, 60%), hsl(207, 82%, 66%));
+}
+
+.language-markdown .token.url>.token.url,
+.language-markdown .token.url-reference.url {
+  color: light-dark(hsl(198, 99%, 37%), hsl(187, 47%, 55%));
+}
+
+.language-markdown .token.blockquote.punctuation,
+.language-markdown .token.hr.punctuation {
+  color: light-dark(hsl(230, 4%, 64%), hsl(220, 10%, 40%));
+  font-style: italic;
+}
+
+.language-markdown .token.code-snippet {
+  color: light-dark(hsl(119, 34%, 47%), hsl(95, 38%, 62%));
+}
+
+.language-markdown .token.bold .token.content {
+  color: light-dark(hsl(35, 99%, 36%), hsl(29, 54%, 61%));
+}
+
+.language-markdown .token.italic .token.content {
+  color: light-dark(hsl(301, 63%, 40%), hsl(286, 60%, 67%));
+}
+
+.language-markdown .token.strike .token.content,
+.language-markdown .token.strike .token.punctuation,
+.language-markdown .token.list.punctuation,
+.language-markdown .token.title.important>.token.punctuation {
+  color: light-dark(hsl(5, 74%, 59%), hsl(355, 65%, 65%));
+}
+
+/* General */
+.token.bold {
+  font-weight: bold;
+}
+
+.token.comment,
+.token.italic {
+  font-style: italic;
+}
+
+.token.entity {
+  cursor: help;
+}
+
+.token.namespace {
+  opacity: 0.8;
+}
+
+/* Plugin overrides */
+/* Selectors should have higher specificity than those in the plugins' default stylesheets */
+
+/* Show Invisibles plugin overrides */
+.token.token.tab:not(:empty):before,
+.token.token.cr:before,
+.token.token.lf:before,
+.token.token.space:before {
+  color: light-dark(hsla(230, 8%, 24%, 0.2), hsla(220, 14%, 71%, 0.15));
+  text-shadow: none;
+}
+
+/* Toolbar plugin overrides */
+/* Space out all buttons and move them away from the right edge of the code block */
+div.code-toolbar>.toolbar.toolbar>.toolbar-item {
+  margin-right: 0.4em;
+}
+
+/* Styling the buttons */
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>button,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>a,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>span {
+  background: light-dark(hsl(230, 1%, 90%), hsl(220, 13%, 26%));
+  color: light-dark(hsl(230, 6%, 44%), hsl(220, 9%, 55%));
+  padding: 0.1em 0.4em;
+  border-radius: 0.3em;
+}
+
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>button:hover,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>button:focus,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>a:hover,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>a:focus,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>span:hover,
+div.code-toolbar>.toolbar.toolbar>.toolbar-item>span:focus {
```

Comment:
**suggestion:** Looks an awful lot like a place where you could just use CSS nesting.